themes/10up-theme/assets/css/frontend/style.css

```diff
@@ -4,25 +4,33 @@
  *
  */
 
+/* Tailwind */
+@import url("tailwind/index.css");
```

Comment:
@Nikki-Jones From what I can see it looks like the idea here is that you want the entire tailwind file included separately so everything else comes after those styles, unless someone chooses to use the `@layer` rule? 

This is actually how we had this initially, but at a point I made an adjustment to what is in the epic now for a few reasons, but the biggest one is it somewhat limits our flexibility with the cascade in a way that the default scaffold does not (since everything is included in a certain order in the scaffold out of the box). 

For instance, if I wanted to always include a class in our bundle, but insert it at the same point as other "component styles" (which allows the Tailwind utilities to take priority still and cleanly override when selector weight is equal) I wouldn't be able to do that because anything I leave out of an `@layer` rule would come after everything else (including tailwind's built-in utilities). This type of thing can cause a utility to not be able to override a component style (picture a button where you need to change 1-2 small things about it and nothing else, so you add 2 utilities with your button class) because the utilities are no longer as specific as they are by default. 

I've only built around 3 sites with WordPress and Tailwind so far so I couldn't say exactly how often this might happen, but it strikes me as a likely possibility we can avoid by just organizing things differently (like we are now). This allows the developer to either include their code at the correct point and always have it be present, or opt-in to using `@layer` based on the situation. 

I'd love to get everyone's take on this and see what we all thing. cc: @Firestorm980 @dainemawer

Reply:
@iansvo as per our slack discussion, take a look with how I've structured things now, let me know what you think!

Reply:
I feel better about this with the cascade for sure. Thanks so much for talking through it with me. 

I would not that I'm not sure we really need to expose an option to change where variants go. But I don't want to just nit pick endlessly. We can iterate on a few things like that once this thing gets in front of people and see if it actually confuses/footguns anyone or not. 

I really appreciate you taking the time to comment these things and explain more of the mentality behind it. I think as we're looking to increase adoption that will be helpful to make this more friendly to new devs. 

Good stuff!